### PR TITLE
Replace NcMultiselect with NcSelect

### PR DESCRIPTION
**Pull request by GretaD**
After the release of nc/vue 8.0.0 NcMultiselect is deprecated and replaced by NcSelect: https://github.com/nextcloud-libraries/nextcloud-vue/releases/tag/v8.0.0

### CODE REVIEW

File: src/components/AccountForm.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
 			:options="{useUrlFragment: false, defaultTabHash: settingsPage ? 'manual' : 'auto'}"
 			cache-lifetime="0"
 			@changed="onModeChanged">
-			<Tab id="auto" key="auto" :name="t('mail', 'Auto')">
+			<Tab id="auto" key="auto" :title="t('mail', 'Auto')">
```
Comment (ChristophWurst) [inline src/components/AccountForm.vue:7]: unrelated?

File: src/components/MailboxThread.vue [hunk 2]
```diff
@@ -25,7 +25,7 @@
 					:open-first="mailbox.specialRole !== 'drafts'" />
 				<template v-else>
 					<div v-show="hasImportantEnvelopes" class="app-content-list-item">
-						<SectionTitle class="important" :name="t('mail', 'Important')" />
+						<SectionTitle class="important" :title="t('mail', 'Important')" />
```
Comment (ChristophWurst) [inline src/components/MailboxThread.vue:28]: unrelated?

**Comment by GretaD** [comment]
i tried to test every component. Please test it before approving because its a bit change and i might have missed something.

File: src/components/SearchMessages.vue [hunk 3]
```diff
@@ -184,20 +184,20 @@
...
 					<div class="modal-inner--container">
-						<NcMultiselect
+						<NcSelect
 							v-if="tags.length > 0"
 							id="tagsId"
 							v-model="selectedTags"
 							class="multiselect-search-tags"
 							:options="tags"
-							label="displayName"
+							label="name"
```
Comment (ChristophWurst) [inline src/components/SearchMessages.vue:193]: why does it change?

Comment (GretaD) [reply src/components/SearchMessages.vue:193]: thats a copy paste issue. it should be as it was

File: src/components/Composer.vue [hunk 4]
```diff
@@ -6,12 +6,12 @@
...
 					id="from"
+					label="name"
 					:value="selectedAlias"
 					:options="aliases"
-					label="name"
-					track-by="selectId"
+					input-id="select-email-input"
 					:searchable="false"
 					:placeholder="t('mail', 'Select account')"
 					:clear-on-select="false"
```
Comment (hamza221) [inline src/components/Composer.vue:17]: ``
```suggestion
					:clear-search-on-select="false"
```
https://vue-select.org/api/props.html#clearsearchonselect

File: src/components/Composer.vue [hunk 5]
```diff
@@ -26,22 +26,22 @@
...
+					label="label"
 					:class="{'opened': !autoLimit}"
 					:options="selectableRecipients"
 					:taggable="true"
-					label="label"
-					track-by="email"
+					input-id="select-email-input"
 					:multiple="true"
 					:placeholder="t('mail', 'Contact or email address …')"
 					:clear-on-select="true"
```
Comment (hamza221) [inline src/components/Composer.vue:47]:
```suggestion
					:clear-search-on-select="true"
```

**Review by hamza221 — CHANGES_REQUESTED** [review]

File: src/components/Composer.vue [hunk 6]
```diff
@@ -6,12 +6,12 @@
 				{{ t('mail', 'From') }}
 			</label>
 			<div class="composer-fields--custom">
-				<Select
+				<NcSelect
```
Comment (GVodyanov) [inline src/components/Composer.vue:9]: image

When clicking on the empty line underneath, an empty select rectangle opens up, don't think this is supposed to happen

Comment (hamza221) [reply src/components/Composer.vue:9]: Yes, composer is still broken

**Review by GVodyanov — COMMENTED** [review]
Graphical issue in composer

**Comment by GretaD** [comment]
> Graphical issue in composer

thank you grigory for the review, this pr is outdated because we are gettting in the #9144
